clients/runtime/src/lib.rs

```diff
@@ -64,41 +67,39 @@ compile_error!("You need to select at least one of the metadata features");
 // All of the parachain features use the same metadata (from Foucoco) for now.
 // We can change this once the spacewalk pallets were added to the runtimes of the other chains as
 // well.
+
 #[cfg_attr(
 	feature = "standalone-metadata",
 	subxt(
 		runtime_metadata_path = "metadata-standalone.scale",
 		derive_for_all_types = "Clone, PartialEq, Eq",
+		substitute_type(path = "sp_core::crypto::AccountId32", with = "crate::AccountId"),
+		substitute_type(path = "spacewalk_primitives::CurrencyId", with = "::subxt::utils::Static<crate::CurrencyId>"),
+		substitute_type(path = "sp_arithmetic::fixed_point::FixedU128", with = "::subxt::utils::Static<crate::FixedU128>"),
 	)
 )]
-#[cfg_attr(
-	feature = "parachain-metadata-pendulum",
-	subxt(
-		runtime_metadata_path = "metadata-parachain-pendulum.scale",
-		derive_for_all_types = "Clone, PartialEq, Eq",
-	)
-)]
-#[cfg_attr(
-	feature = "parachain-metadata-amplitude",
-	subxt(
-		runtime_metadata_path = "metadata-parachain-amplitude.scale",
-		derive_for_all_types = "Clone, PartialEq, Eq",
-	)
-)]
-#[cfg_attr(
-	feature = "parachain-metadata-foucoco",
-	subxt(
-		runtime_metadata_path = "metadata-parachain-foucoco.scale",
-		derive_for_all_types = "Clone, PartialEq, Eq",
-	)
-)]
-pub mod metadata {
-	#[subxt(substitute_type = "sp_core::crypto::AccountId32")]
-	use crate::AccountId;
-	#[subxt(substitute_type = "spacewalk_primitives::CurrencyId")]
-	use crate::CurrencyId;
-	#[subxt(substitute_type = "sp_arithmetic::fixed_point::FixedU128")]
-	use crate::FixedU128;
+// #[cfg_attr(
```

Comment:
Is this note required anymore? In that case can we remove the comments?

Reply:
Apparently with the new `subxt` version, the macro will fail when `all--features` flag is passed. 
This didn't happen before.

 I left this commented to discuss this because as the [previous comment](https://github.com/pendulum-chain/spacewalk/pull/536/files/4d27e969bbe1502c6a90e2cd90d7345602238c14#diff-19f5b8d58a7b3276f119607fd62317f9fde6fe5f4d88e8b639b7404fd661c9c2R68) above said, we are always using foucoco metadata. I imagine this was because this was the last call to [the](https://github.com/pendulum-chain/spacewalk/pull/536/files/4d27e969bbe1502c6a90e2cd90d7345602238c14#diff-19f5b8d58a7b3276f119607fd62317f9fde6fe5f4d88e8b639b7404fd661c9c2R95) macros. So with this changes we would be using only the `standalone` metadata for every case. I am not sure if this is correct, or we still need to differentiate. In the later case, we would need to tweak a bit more the conditionals.

Reply:
@ebma what do you think?

Reply:
I think the comment 
```
// All of the parachain features use the same metadata (from Foucoco) for now.
// We can change this once the spacewalk pallets were added to the runtimes of the other chains as
// well.
```
is stale and needs to be removed. I added it at a time where Spacewalk was only live on Foucoco, so that was the only metadata available. 

Let's try to figure out why exactly the subxt macro fails for `--all-features`. But we definitely need to keep differentiating the .scale files per runtime otherwise the vaults will probably not be able to connect to the respective network anymore.

Reply:
I think the reason it fails is because when compiling with `--all-features`, it tries to apply the macro multiple times. Not sure why this was not an issue before `subxt` update. 

So I think this [changes](https://github.com/pendulum-chain/spacewalk/pull/536/commits/4409eef8a9870325ab9084a224d1232cb8508060) should solve this. I added an extra feature `all-features` that is not intended to do anything but be activated when `--all-features` is used. Then we can differentiate and select only one metadata, in this case standalone. As far as I know selecting all features in only used during testing right?